talk/owt/BUILD.gn

```diff
@@ -222,9 +222,7 @@ static_library("owt_sdk_base") {
     defines += [ "WEBRTC_USE_X11" ]
   }
 
-  if (!rtc_use_h265) {
-    defines += [ "DISABLE_H265" ]
-  } else {
+  if (rtc_use_h265) {
     defines += [ "OWT_USE_H265" ]
```

Comment:
Do you need this def?

Reply:
Some legacy code was using OWT_USE_H265. Renamed to WEBRTC_USE_H265.